### PR TITLE
fix: expose region variable

### PR DESCRIPTION
**Pull request by iamar7**
### Description

Resolves: https://github.com/terraform-ibm-modules/terraform-ibm-event-streams/issues/250

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [x] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: modules/fscloud/variables.tf [hunk 1]
```diff
@@ -16,6 +16,12 @@ variable "es_name" {
   type        = string
 }
 
+variable "region" {
+  type        = string
+  description = "IBM Cloud region where event streams will be created"
```
Comment (SirSpidey) [inline modules/fscloud/variables.tf:21]: Any reason why this is different from solutions/standard/variables.tf?

>   description = "The region where the IBM Event Streams for IBM Cloud are created and all resources provisioned by this solution are located."

File: solutions/standard/variables.tf [hunk 2]
```diff
@@ -22,7 +22,7 @@ variable "es_name" {
 
 variable "region" {
   type        = string
-  description = "Region to provision all resources created by this solution."
+  description = "The region where the IBM Event Stream will be created and all resources provisioned by this solution will be located.."
```
Comment (SirSpidey) [inline solutions/standard/variables.tf:25]:
```suggestion
  description = "The region where the IBM Event Streams for IBM Cloud are created and all resources provisioned by this solution are located."
```

**Review by SirSpidey — CHANGES_REQUESTED** [review]
syncing the descriptions

File: modules/fscloud/variables.tf (hunk 1 above)
Comment (iamar7) [reply modules/fscloud/variables.tf]: I've made the variable description consistent.

**Review by Ak-sky — APPROVED** [review]

**Comment by Ak-sky** [comment]
wait for 30 min for next trigger-
```
Error: fsCloudSolution 2024-06-11T09:16:29Z logger.go:66: │ Error: [ERROR] Error when creating resource instance: Please contact the Service Provider for this error. [409, Conflict] You have an active provision that is less than 30 minutes old. Please wait until either 30 minutes have passed since your previous provision, or until the previous provision has completed. with resp code: {
TestRunfsCloudSolution 2024-06-11T09:16:29Z logger.go:66: │     "StatusCode": 422,
```

**Comment by iamar7** [comment]
@SirSpidey Can you approve this PR?

**Review by SirSpidey — APPROVED** [review]
👍

**Pull request merged**